docs/QUICKSTART.md

```diff
@@ -124,9 +124,13 @@ TrustedUserCAKeys=/etc/ssh/hallow_cas
 AuthorizedPrincipalsFile=/etc/ssh/principals/%u
 ```
 
-Write the key from `hallow-ca.pub` into `/etc/ssh/hallow_cas` on the remote
+Write the key from `hallow-ca.pub` to `/etc/ssh/hallow_cas` on the remote
 machine.
 
+```
+$ sudo echo "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFvuBGdFLPNRg+xZkGfQ5u9V3FD6etx0cz0fx6HkjzAvZ0W/FF4HYZPsCkLpsJhjaRfF1Nm9mNXiyaHsrkfaKgQ=" > /etc/ssh/hallow_cas
```

Comment:
`sudo echo` won't do what you want :-)

```suggestion
$ echo "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFvuBGdFLPNRg+xZkGfQ5u9V3FD6etx0cz0fx6HkjzAvZ0W/FF4HYZPsCkLpsJhjaRfF1Nm9mNXiyaHsrkfaKgQ=" | sudo tee /etc/ssh/hallow_cas
```

Reply:
I agree, but just being consistent with the rest of the documentation, shall I change this everywhere?